scipy/stats/_continuous_distns.py

```diff
@@ -647,10 +648,10 @@ class burr_gen(rv_continuous):
 
     """
     def _pdf(self, x, c, d):
-        return c*d*(x**(-c-1.0))*((1+x**(-c*1.0))**(-d-1.0))
+        return c*d*(x**(-c-1.0))*((1+x**(-c))**(-d-1.0))
 
     def _cdf(self, x, c, d):
-        return (1+x**(-c*1.0))**(-d**1.0)
+        return (1+x**(-c))**(-d)
```

Comment:
Oh okay, just thought it would be better to change even in that code. But got the point !